### DISCUSSION TITLE
Camera Storage in system/storage tab - Help

### DISCUSSION THREAD

**Discussion by akcasoy** [blakeblackshear/frigate#14519] [Ask A Question]
I have a proxmox installation and assigned 32GB space for Frigate.
This is how my Proxmox LXC looks like:

<img width="570" alt="Bildschirmfoto 2024-10-23 um 00 24 23" src="https://github.com/user-attachments/assets/94451f14-6294-47db-8fda-bb77c125e319">

And this is what Frigate Storage Tab shows me:
<img width="1179" alt="Bildschirmfoto 2024-10-23 um 00 20 12" src="https://github.com/user-attachments/assets/4afedffc-1153-41d5-a091-db55c4f99214">

What are those "Camera Storage"s  on the lowest part? What is 527.70 MiB? And why is one of the cameras nearly full there?

**Comment by NickM-27** [comment]
this just shows how much of the currently used storage is used by each camera. This is made more clear in the next version of frigate

**Reply by akcasoy** [reply]
Thank you Nick :) I have realized that just after i posted it. 👍
